src/dataSource.ts

```diff
@@ -2,13 +2,23 @@ import { DataSource } from 'typeorm';
 
 import { Config } from './utils/config';
 
-export const dataSource = new DataSource({
-  type: 'postgres',
-  host: Config.database.host,
-  port: Config.database.port,
-  username: Config.database.username,
-  password: Config.database.password,
-  database: Config.database.name,
-  synchronize: false,
-  logging: false,
-});
+let dataSource: DataSource;
+if (process.env.NODE_ENV === 'test') {
+  dataSource = new DataSource({
+    type: 'sqlite',
+    database: Config.database.name
```

Comment:
This DBName is not the same as DBName in pg.
Add a new config, called `filename` or `path` and refactor this.

Reply:
You've already done it. You just have to connect to the DB in the proper way.